src/usr/local/www/services_dhcp.php

```diff
@@ -868,12 +886,19 @@ function build_pooltable() {
 	$pconfig['ignorebootp']
 ));
 
-$section->addInput(new Form_Checkbox(
+$section->addInput(new Form_Select(
 	'denyunknown',
 	'Deny unknown clients',
-	'Only the clients defined below will get DHCP leases from this server.',
-	$pconfig['denyunknown']
-));
+	$pconfig['denyunknown'],
+	array(
+		"disabled" => "Allow Any (any MACs, including any not listed anywhere)",
```

Comment:
Not sure this needs the parenthetical part. `Allow all clients` gets the point across clearly and concisely.